webapp/src/constants.ts

```diff
@@ -10,6 +10,7 @@ export const DEFAULT_RING_SOUND = 'Calm';
 export const CALL_START_POST_TYPE = 'custom_calls';
 export const CALL_RECORDING_POST_TYPE = 'custom_calls_recording';
 export const CALL_TRANSCRIPTION_POST_TYPE = 'custom_calls_transcription';
+export const LIVE_CAPTION_TIMEOUT = 5000;
```

Comment:
Do you think this is generally enough? I suppose it depends on what's the maximum output length of the transcriber.

Reply:
This timer only affects a caption that hasn't had any changes -- so the caption will always be there if there's updates or new lines etc, and only once the participant stops speaking will the timeout remove the caption (after 5 seconds of not speaking). I've found when watching the load test that this leaves a natural amount of time to read the caption. We can tune it to whatever of course.